.github/workflows/package-size-check.js

```diff
@@ -42,7 +42,29 @@ console.log(`features: [ ${features.join(', ')} ]`);
         },
         wasmCompressionMode: 'brotli',
         inlineEnum: true,
-        mangleProperties: true,
+        mangleProperties: {
+            mangleList: [
+                'UITransform._sortSiblings',
+                'UITransform._cleanChangeMap',
+                'Node._findComponents',
+                'Node._findChildComponent',
+                'Node._findChildComponents',
+                'Node.idGenerator',
+                'Node._stacks',
+                'Node._stackId',
+                'Node._setScene',
+                'EffectAsset._layoutValid',
+                'EffectAsset._effects',
+                'ReflectionProbe.DEFAULT_CUBE_SIZE',
+                'ReflectionProbe.DEFAULT_PLANER_SIZE',
+                'WebGLDeviceManager.setInstance',
+                'WebGL2DeviceManager.setInstance',
+                'CanvasPool',
```

Comment:
Why need to add the list?

Reply:
They will be add to default mangle config list too.
They are some protected properties that 99% people will not use.
If the 1% people want to use it ,just remove some lines in `engine-mangle-config.json`

Reply:
As it is to mangle engine variables and functions, why not add `@mangle` in codes.

Reply:
We could.
Just to put the default mangle configuration in code by `@mangle` or put them to `mangleList` in  template configuration.
As we discuss before, protected properties of `Component` should not been mangled has already been set in `dontMangleList` in the template configuration.

Reply:
Ok, got it.